modules/tinymce/src/plugins/link/main/ts/ui/Controls.ts

```diff
@@ -79,6 +79,21 @@ const setupContextToolbars = (editor: Editor) => {
     return Fun.noop;
   };
 
+  /**
```

Comment:
```suggestion
  /*
```
This just seems to be a regular comment and not a API type doc string so we should only have a single asterisk.

Reply:
This could also just be converted to using single line comments, which is what is more commonly used for this in the current code base 🤷

Reply:
Just habit, I guess. Even if a method doc comment doesn't include details of function arguments and return type it will still show up as autocomplete documentation.